package-lock.json

```diff
@@ -1,12048 +1,6 @@
 {
-  "name": "Lombiq.Gulp.Extensions",
```

Comment:
Ez nekem downgrade-nek tűnik, és meg is változik:
![image](https://user-images.githubusercontent.com/1976647/133534349-cb0ae0cd-7642-4d38-a44c-8d50baa1ca2f.png)